enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -206,6 +207,10 @@
   (mdb/setup-db!)
   (check-premium-token!)
   (t2/select User) ;; TODO -- why??? [editor's note: this comment originally from Cam]
+  (let [f (io/file path)]
+    (.mkdirs f)
+    (when-not (.canWrite f)
+      (throw (ex-info (format "Destination path is not writeable: %s" path) {:filename path}))))
```

Comment:
This is just so that user doesn't need to wait until the extraction process started.